workspaces/netlify-cms-backend-github/src/API.ts

```diff
@@ -35,6 +34,8 @@ import type {
 import type { Semaphore } from 'semaphore';
 import type { Octokit } from '@octokit/rest';
 
+const CMS_BRANCH_PREFIX = `${import.meta.env.VITE_GIT_BRANCH_NAME}-cms`;
```

Comment:
@lorcan-codes  you need to also patch this two functions https://github.com/decaporg/decap-cms/blob/master/packages/netlify-cms-lib-util/src/APIUtils.ts#L32-L38